spec/jobs/async_batch_update_job_spec.rb

```diff
@@ -7,12 +7,14 @@
     include_context :init_destination, id: 2, initial_rate: 0.5
     include_context :init_destination, id: 3, initial_rate: 0.7
 
-    subject { described_class.new(model_class, sql_query, changes).perform}
+    subject { described_class.new(model_class, sql_query, changes, who_is).perform }
 
     before :each do
       stub_const('AsyncBatchUpdateJob::BATCH_SIZE', 2)
     end
 
+    let(:who_is) { create :admin_user }
+
```

Comment:
@stekatk same about ```whoi_is```. that I mentioned above.
Also I believe tests should be added to check who_is was  passed to Audit logs